controller/volume_controller_test.go

```diff
@@ -128,20 +128,22 @@ func (s *TestSuite) TestVolumeLifeCycle(c *C) {
 	}
 	tc.expectVolume.Status.State = longhorn.VolumeStateCreating
 	tc.expectVolume.Status.CurrentImage = tc.volume.Spec.EngineImage
-	tc.expectVolume.Status.Robustness = longhorn.VolumeRobustnessUnknown
 	tc.expectVolume.Status.Conditions = setVolumeConditionWithoutTimestamp(tc.expectVolume.Status.Conditions,
 		longhorn.VolumeConditionTypeScheduled, longhorn.ConditionStatusFalse, longhorn.VolumeConditionReasonReplicaSchedulingFailure, longhorn.ErrorReplicaScheduleNodeUnavailable)
 	testCases["volume create - replica scheduling failure"] = tc
 
 	// after creation, volume in detached state
 	tc = generateVolumeTestCaseTemplate()
+	tc.volume.Status.State = longhorn.VolumeStateDetaching
```

Comment:
NIT: Do we need to add a case that explain how the volume state is updated from `creating` to `detaching`?

Reply:
Good idea. Added. Thank you!